examples/graphene_apps/cppopenvino/ov_workload/src/openvino.py

```diff
@@ -60,7 +60,7 @@ def execute(self, in_data_array):
         Parameters :
             in_data_array: Input data array containing data in plain bytes
         Returns :
-            status as boolean and output result in bytes.
+            status as boolean and output result in string.
```

Comment:
Why is the function prototype is changed ? Byte sis more generic than string. Not all bytes values can be converted to a valid UTF-8 string

Reply:
Reverted back to old prototype

Reply:
Reverted back to old prototype